dvc/repo/params/show.py

```diff
@@ -50,17 +51,18 @@ def _read_params(repo, configs, rev):
     return res
 
 
-def _add_vars(repo, configs, params):
+def _add_vars(repo, params):
```

Comment:
I think that maybe we should change the naming, too. Now we are not adding anything. How about `_stage_vars`, to stress the fact that those are params loaded from the stage?